ark/utils/plot_utils.py

```diff
@@ -356,3 +357,100 @@ def create_overlay(fov, segmentation_dir, data_dir,
         rescaled[alternate_contour_mask > 0, 1:] = 0
 
     return rescaled
+
+
+def create_mantis_project(mantis_project_path: Union[str, pathlib.Path],
+                          img_data_path: Union[str, pathlib.Path],
+                          mask_output_dir: Union[str, pathlib.Path],
+                          mapping_path: Union[str, pathlib.Path],
+                          seg_dir: Union[str, pathlib.Path],
+                          mask_suffix: str = "_mask", img_sub_folder: str = "normalized"):
+    """Creates a mantis project directory so that it can be opened by the mantis viewer.
+    Copies fovs, segmentation files, masks, and mapping csv's into a new directory structure.
+    Here is how the contents of the mantis project folder will look like.
+
+    ```{code-block} sh
+    mantis_project
```

Comment:
This should go in the readme as well, since that's where people are most likely to see it